docker/test-app/test_app/urls.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2022 UW-IT, University of Washington
+# SPDX-License-Identifier: Apache-2.0
+
+from django.urls import path
+from django.views.decorators.csrf import csrf_exempt
+from .views import get_spots, get_buildings, get_spots_by_search, \
+     SpotView, ItemImageView, SpotImageView
+
+urlpatterns = [
+    path("spot/<int:spot_id>/image/<int:img_id>",
+         csrf_exempt(SpotImageView.as_view())),
+    path("spot/<int:spot_id>/image", csrf_exempt(SpotImageView.as_view())),
```

Comment:
this was done to allow gets, puts, deletes, and posts from the same class since posts require different URLs